sdcm/cluster.py

```diff
@@ -2556,23 +2573,26 @@ def install_scylla_monitoring_prereqs(self, node):
                 pip install pyyaml
             """)
         elif node.is_debian():
+            node.remoter.run('sudo apt-key adv --keyserver keyserver.ubuntu.com --recv-keys F76221572C52609D', retry=3)
+            node.remoter.run(cmd="sudo apt-get update", ignore_status=True)
+            node.remoter.run(cmd="sudo apt-get dist-upgrade -y")
+            node.remoter.run(
+                cmd="sudo echo 'deb https://apt.dockerproject.org/repo debian-jessie main' | sudo tee /etc/apt/sources.list.d/docker.list")
+            node.remoter.run(cmd="sudo apt-get update", ignore_status=True)
+            node.remoter.run(cmd="sudo apt-get install docker-engine -y --force-yes")
+            node.remoter.run(cmd="sudo apt-get install -y curl")
+            node.remoter.run(cmd="sudo apt-get install software-properties-common -y")
+            node.remoter.run(cmd="sudo apt-get update", ignore_status=True)
+            node.remoter.run(cmd="sudo apt-get install -y python-setuptools wget")
```

Comment:
why do you install those one by one? please aggregate as in `prereqs_script`

Reply:
because:
1. after the archive of Jesse, each command is prone to fail separately, so this way it is more clear and accurate.
2. when an 'apt-get update' command is required, it is currently separated since always returns some errors (due to archiving) so it must be issued separately, using the "ignore_status" flag.